modules/ducktests/src/main/java/org/apache/ignite/internal/ducktest/utils/IgniteAwareApplicationService.java

```diff
@@ -60,6 +61,11 @@ public static void main(String[] args) throws Exception {
 
         app.cfgPath = cfgPath;
 
+        String connStr = jsonNode.get("thin_client_connection").asText();
+
+        if (connStr != null && !connStr.isEmpty())
```

Comment:
We should have a proper switch here.
`startIgnite` should be replaced with Enum [Node, ThinClient, None]
The second param in this case should contain configuration for ThinClient if the `ThinClient` case selected.

Reply:
Additional reflection suggests we may want to test the local Thin Client case, so, enum is not an option here.
But, since `IgniteClient extends AutoCloseable` we must wrap the current try-with-resources section with an additional one.